presto-main/src/test/java/com/facebook/presto/execution/scheduler/TestSourcePartitionedScheduler.java

```diff
@@ -310,7 +314,7 @@ public void testScheduleSlowSplitSource()
     public void testNoNodes()
     {
         try {
-            NodeTaskMap nodeTaskMap = new NodeTaskMap(finalizerService);
+
```

Comment:
nit : remove new line

Reply:
Done. Thank you very much for the feedback!